settings.gradle

```diff
@@ -1,8 +1,11 @@
 rootProject.name = 'LegacyNativeExample'
 
+
 include 'static_lib'
 include 'subsystem_a:util_1'
-include 'subsystem_a:win_mfc_a'
+if(org.gradle.nativeplatform.platform.internal.DefaultNativePlatform.currentOperatingSystem.isWindows()) {
+    include 'subsystem_a:win_mfc_a'
+}
```

Comment:
You should prefer fixing the consumer then removing the project from the build.

Reply:
I couldn't make this work, at least not simply, as I didn't have another output to place for the installer, so it failed because then the Source directory didn't exist for the zip command.   something for later, I suspect.